backend/common/src/main/java/ai/verta/modeldb/common/config/GrpcServerConfig.java

```diff
@@ -2,6 +2,7 @@
 
 public class GrpcServerConfig {
   public int port;
+  public int threadCount = 8;
```

Comment:
Why here 8 is the ideal one??

Reply:
Just a number that isn't zero for backwards compatibility with the existing config =) There's no "ideal" one for the number of threads.